pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -65,6 +68,33 @@ var (
 	}
 )
 
+var vendorRegistry = map[uint32]string{
```

Comment:
Would be useful to have a comment indicating where these values comes from to make it easier to find out where to look for new manufacturers in the future.